fusion_bench/utils/rich_utils.py

```
@@ -73,12 +100,54 @@ def print_bordered(message, title=None, style="blue", code_style=None):
     else:
         content = Text(message)
 
-    panel = Panel(content, title=title, border_style=style)
-    print(panel)
+    panel = Panel(content, title=title, border_style=style, expand=expand)
+    print_fn(panel)
+
+
+def print_code(
+    message,
+    title=None,
```

Ruff (0.14.8)

Unused function argument: `title`

(ARG001)